jwt/jwt.go

```diff
@@ -95,7 +95,7 @@ func Verify(req *http.Request, keyServer keyserver.Reader, nonceVerifier noncest
 
 	if token == "" {
 		// Not found anywhere
-		return nil, &authRequiredError{"No JWT found", "http://" + req.Host + req.URL.String()}
```

Comment:
I guess it won't work. I googled possible solution: 
```
isHTTPS := req.TLS != nil
```

I guess @mshaposhnik is also trying to fix it. Please sync

Reply:
OK, no problem. I'll let him come back to me, if he needs me to spend more time on it.

Reply:
Ah, OK, you mean that the initial request may have the `http` scheme, but in fact be a TLS request so that we should redirect it to `https` ?

Reply:
@davidfestal I mean that it may be just an empty string for incoming requests
https://stackoverflow.com/questions/40826664/get-scheme-of-the-current-request-url

Reply:
Oh, OK, sorry. I was not aware of this GO behavior.

Reply:
No problem. I was surprised with such behavior as well.